.travis.yml

```diff
@@ -1,4 +1,9 @@
 language: rust
 rust:
+  - stable
+  - beta
   - nightly
-
+matrix:
+  allow_failures:
+    - rust: nightly
+script: cargo test
```

Comment:
It looks like `cargo test` is already part of the default Travis Rust configuration: https://docs.travis-ci.com/user/languages/rust/
Is this definitely needed?